<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboBoxFont</class>
 <widget class="QWidget" name="ComboBoxFont">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QComboBox" name="fontSizeComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QFontComboBox" name="fontTypeComboBox"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="fontExampleQLabel">
        <property name="text">
         <string>The quick brown fox jumps over the lazy dog.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fontColorComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fontStyleComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
